<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMainWindow</class>
 <widget class="QWidget" name="AdminMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="showInfoButton">
      <property name="text">
       <string>نمایش اطلاعات</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addStudentButton">
      <property name="text">
       <string>ثبت دانشجو</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addMasterButton">
      <property name="text">
       <string>ثبت استاد</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addCourseButton">
      <property name="text">
       <string>افزودن درس</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
